pypowerwall/tedapi/__init__.py

```diff
@@ -375,18 +392,29 @@ def calculate_dc_power(V, I):
             i = 0
             c = c + 1
             for ct in n['dataRead'] or {}:
+                # Only show if we have a meter configuration and cts[i] is true
+                cts_bool = lookup(meter_config, [sn, 'cts'])
+                if isinstance(cts_bool, list) and i < len(cts_bool):
+                    if not cts_bool[i]:
+                        # Skip this CT
+                        i = i + 1
+                        continue
+                factor = lookup(meter_config, [sn, 'real_power_scale_factor']) or 1
                 device = f"NEURIO_CT{i}_"
-                cts[device + "InstRealPower"] = lookup(ct, ['realPowerW'])
+                cts[device + "InstRealPower"] = lookup(ct, ['realPowerW']) * factor
```

Comment:
I'm not sure if this is what the original vitals did, but it appears to be the case. The CTs are usually only applied to one line (where there are 2 or 3 lines/phases) and the config is set to double or triple the value in those cases. For my SolarRGM, it has a value of 2.